src/test/ui/suggestions/type-ascription-instead-of-initializer.stderr

```diff
@@ -0,0 +1,17 @@
+error: expected type, found `10`
+  --> $DIR/type-ascription-instead-of-initializer.rs:12:31
+   |
+12 |     let x: Vec::with_capacity(10, 20);
+   |         --                    ^^
+   |         ||
+   |         |help: did you mean assign here?: `=`
```

Comment:
The `?:` looks funny, don't we have some convention for dealing with that? cc @oli-obk

Reply:
Yea: https://github.com/rust-lang/rust/blob/9ae6ed78acf9dc865e2300a7db556389eed9692d/src/librustc_errors/diagnostic.rs#L231-L236